spec/controllers/entries_controller_spec.rb

```diff
@@ -27,5 +27,19 @@
       get :show, params: {list_id: 1, id: 1}
       expect(assigns(:list)).not_to eq(list)
     end
+
+    it "returns nil when finding a list that does not exist" do
+      user = User.create!(id: 1, username: "Squid", email: "s@s.com", first_name: "Sid", last_name: "Monty")
+      list = List.create!(id: 2, title: "heyo", user: user)
+      get :show, params: {list_id: 1, id: 1}
```

Comment:
So while you implemented the test, we want to ensure that the List we're searching for doesn't exist.

Since you are searching for list_id: 1, you should have a:
expect(List.find_by_id(1).to be_nil

Reply:
@Nakirendral Please change this to the following:
```ruby
  it "returns nil when finding a list that does not exist" do
      user = User.create!(id: 1, username: "Squid", email: "s@s.com", first_name: "Sid", last_name: "Monty")
      list = List.create!(id: 2, title: "heyo", user: user)
      
      expect(List.find_by_id(1).to be_nil
      get :show, params: {list_id: 1, id: 1}
      
      expect(assigns(:list)).to eq nil
    end
```
You can have two expectations in the same test as long as it is clear what you are testing.